### ISSUE TITLE
Intermittent Connection pool shut down

### ISSUE THREAD

**Issue by javiermartinezruizvm** [twilio/twilio-java#628]
### Issue Summary
When sending an SMS using the Twilio SDK 7.3.0, intermittently, I'm receives an **java.lang.IllegalStateException: Connection pool shut down**

### Steps to Reproduce
1. When sending an SMS, intermittently, I'm receives an **java.lang.IllegalStateException: Connection pool shut down**

### Code Snippet
```java
Twilio.init(subAccountSid, secret);
final String messageText = getMessageText(message);
            final MessageCreator messageCreator = Message.creator(subAccountSid, new PhoneNumber(smsTo), new PhoneNumber(smsFrom), messageText);
            callsMeter.mark();
            final Message sentMessage = messageCreator.create();
```

### Exception/Log
```
java.lang.IllegalStateException: Connection pool shut down
	at org.apache.http.util.Asserts.check(Asserts.java:34)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.requestConnection(PoolingHttpClientConnectionManager.java:269)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:176)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at com.twilio.http.NetworkHttpClient.makeRequest(NetworkHttpClient.java:82)
	at com.twilio.http.HttpClient.reliableRequest(HttpClient.java:38)
	at com.twilio.http.HttpClient.reliableRequest(HttpClient.java:22)
	at com.twilio.http.TwilioRestClient.request(TwilioRestClient.java:42)
	at com.twilio.rest.api.v2010.account.MessageCreator.create(MessageCreator.java:307)
	at com.twilio.rest.api.v2010.account.MessageCreator.create(MessageCreator.java:25)
	at com.twilio.base.Creator.create(Creator.java:45)
```

### Technical details:
* twilio-java version: 7.3.0
* java version: java 11.0.9 2020-10-20 LTS

**Comment by shwetha-manvinkurke** [comment]
@javiermartinezruizvm The version you mention was released 4.5 years ago. Would you be able to upgrade to the latest version and get back to us if you see the same issue?

**Comment by shwetha-manvinkurke** [comment]
Closing due to inactivity. Please reopen if you still face the issue.
